llvm/lib/Transforms/Vectorize/VPRecipeBuilder.h

```diff
@@ -88,8 +88,7 @@ class VPRecipeBuilder {
 
   /// The set of reduction exit instructions that will be scaled to
   /// a smaller VF via partial reductions, paired with the scaling factor.
-  DenseMap<const Instruction *, std::pair<PartialReductionChain, unsigned>>
-      ScaledReductionExitInstrs;
+  DenseMap<const Instruction *, unsigned> ScaledReductionExitInstrs;
```

Comment:
Should the name and comment also be updated now, this now just maps scaled reduction exit instructions to their scaling factors?

Reply:
Done. I've tried my best. :)